src/components/beacon-plus/BiosamplesDataTable.js

```diff
@@ -12,7 +12,17 @@ export default function BiosamplesDataTable({ url }) {
     () => [
       {
         Header: "Id",
-        accessor: "id"
+        accessor: "id",
+        // eslint-disable-next-line react/display-name
+        Cell: (cellInfo) => (
+          <a
+            href={`https://info.progenetix.org/biosample-details.html?datasetIds=${cellInfo.row.values.project_id}&id=${cellInfo.value}`}
```

Comment:
There's a misunderstanding - the `project_id` is just an occasional; value; for the link the `datasetid` should be used - so you have to import it into the function, I guess.

Reply:
(And we can remove the `project_id` column; not that important).